src/psyclone/tests/parse/algorithm_test.py

```diff
@@ -53,7 +53,7 @@
 from psyclone.errors import InternalError
 
 
-ParserFactory().create()
+ParserFactory().create(std="f2008")
```

Comment:
The fixture didn't work because it seems the import statement needs the Factory constructed to find the elements to import.
But I added a comment and could delete some repeated ParserFactory call inside the tests.